_i18n/en.yml

```diff
@@ -501,6 +501,11 @@ faq:
   avuln: >
     The Monero community has created a series of videos called "Breaking Monero", where potential Monero vulnerabilities are explored and discussed. There are 14 videos, with each exploring a different subject. Check out <a href="https://www.monerooutreach.org/breaking-monero/">the videos on monerooutreach.org</a>.
   vulnspotify: Available on Spotify as podcast
+  qasicresistance: "What is ASIC resistance? Why is it important?"
+  aasicresistance: ASICs are basically special computers created to do only one job, contrary to normal computers, which are made for multiple purposes. This characteristic makes ASICs very efficient for @mining.
```

Comment:
```suggestion
  aasicresistance: ASICs are basically special computers created to do only one job, contrary to normal computers, which are made for general purposes. This characteristic makes ASICs very efficient for @mining.
```

I think general purpose is the correct word here, as theoretically you can create ASICs that are for multiple purposes.